Library/OcConsoleLib/TextOutputBuiltin.c

```diff
@@ -894,7 +908,13 @@ ConsoleControlSetMode (
   IN EFI_CONSOLE_CONTROL_SCREEN_MODE  Mode
   )
 {
-  mConsoleMode = Mode;
+  if (mConsoleMode != Mode) {
+    mConsoleMode = Mode;
+    if (mConsoleMode == EfiConsoleControlScreenText) {
```

Comment:
Don’t bother changing it, but I would always use the local variable in such cases to not rely on compiler optimizations